.github/workflows/task_runner_e2e_wo_mtls.yml

```diff
@@ -0,0 +1,93 @@
+#---------------------------------------------------------------------------
+# Workflow to run Task Runner end to end tests with non TLS client auth
+# Authors - Noopur, Payal Chaurasiya
+#---------------------------------------------------------------------------
+name: Task Runner E2E W/o mTLS
+
+on:
+  workflow_dispatch:
+    inputs:
+      num_rounds:
+        description: 'Number of rounds to train'
+        required: false
+        default: "5"
+        type: string
+      num_collaborators:
+        description: 'Number of collaborators'
+        required: false
+        default: "2"
+        type: string
+
+permissions:
+  contents: read
+
+# Environment variables common for all the jobs
+env:
+  NUM_ROUNDS: ${{ inputs.num_rounds || '5' }}
+  NUM_COLLABORATORS: ${{ inputs.num_collaborators || '2' }}
+
+jobs:
+  test_run:
+    name: tr_non_tls
```

Comment:
There are actually several possible combinations: non-TLS, TLS (server auth only), and mTLS (with client auth). As you cover those cases in future PRs, it could make sense to include those in the matrix, rather than creating a job per case.

Reply:
I have kept the non-TLS as well in the same workflow now under a separate job.

For non-TLS, the matrix will have single combination of torch_cnn_mnist and python 3.10 for now. If required, this can be extended to other models and python versions in future.